crate/operator/restore_backup.py

```diff
@@ -871,16 +879,15 @@ async def handle(  # type: ignore
                 get_host(core, namespace, name),
             )
             conn_factory = connection_factory(host, password)
-            async with conn_factory() as conn:
-                async with conn.cursor() as cursor:
```

Comment:
Could have also set the timeout globally here perhaps? Would have made a smaller diff! :)

Reply:
yes, but then the timeout would be increased for everything in `validate_restore_complete()` like querying `sys.shards` etc. not sure if we want that? 

The diff should become smaller once I merge the other branch into master - I just needed that code as a base here :)

Reply:
I don't think it matters that much, the one minute timeout is a bit arbitrary, same as 2 minutes! Up to you entirely, I already approved it.